sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -129,12 +145,29 @@ export class GCNotifyActionsService {
     const notificationsIds = await this.notificationService.saveNotifications(
       notificationsToSend,
       auditUserId,
-      entityManager,
+      options?.entityManager,
     );
 
-    // TODO: Temporary code to be removed once queue/schedulers are in place.
-    for (const id of notificationsIds) {
-      await this.notificationService.sendEmailNotification(id);
+    if (!options?.notificationsDelayed) {
```

Comment:
what happens when `notificationsDelayed` is true

Reply:
The message is saved to the DB only. The expectation is that there will be a job running to take all the ones that were never sent and try to send them again.